projects/wave-core-new/src/lib/time/time.model.ts

```diff
@@ -165,6 +165,13 @@ export class Time implements ToDict<TimeIntervalDict> {
         return !!this.start && !!this.end && this.start.isValid() && this.end.isValid() && this.start <= this.end;
     }
 
+    /**
+     * Checks if `this` value is smaller than `other` value on both `start` and `end` values.
+     */
+    isSmallerThan(other: Time): boolean {
```

Comment:
maybe a better name would be `before`?

Reply:
[e78b649](https://github.com/geo-engine/geoengine-ui/pull/404/commits/e78b649f06ab2cd13d64a0195a2975d86461c068)